drivers/overlay/ov_network.go

```diff
@@ -781,15 +781,10 @@ func (n *network) Value() []byte {
 		netJSON = append(netJSON, sj)
 	}
 
-	b, err := json.Marshal(netJSON)
-	if err != nil {
-		return []byte{}
-	}
-
```

Comment:
Just checked that is simply unused code. Look at line: https://github.com/docker/libnetwork/blob/master/drivers/overlay/ov_network.go#L883 b is redefined without being used, and here https://github.com/docker/libnetwork/blob/master/drivers/overlay/ov_network.go#L881 netJSON is assigned in the non Marshalled version